versioned/persist/mongodb/pom.xml

```diff
@@ -60,5 +60,11 @@
       <artifactId>de.flapdoodle.embed.mongo</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.jnr</groupId>
+      <artifactId>jnr-unixsocket</artifactId>
+      <version>0.18</version>
```

Comment:
should this maybe go into the `dependencyManagement` section of the main `pom.xml`?